clickhouse_migrate/runner.py

```diff
@@ -13,10 +15,16 @@ def check_positive_int(value):
 
 
 async def _run():
+    environ = {
+        **dotenv_values(".env"),
+        **dotenv_values(".local.env"),
```

Comment:
1. Add support for a configuration file (`.chmigrate`), the path to which can be set using a command line parameter (`--config`)
2. By default use '.env', but also add option `dotenv` to configuration file

Reply:
Add options:
`--env` - environments variables filename
`--dsn` - configure clickhouse dsn
`--migration-path` - configure migration path
`--migration-table` - configure migration table